nipype/algorithms/confounds.py

```diff
@@ -794,6 +835,27 @@ def is_outlier(points, thresh=3.5):
     return timepoints_to_discard
 
 
+def cosine_filter(data, timestep, remove_mean=False, axis=-1):
+    datashape = data.shape
+    timepoints = datashape[axis]
+
+    data = data.reshape((-1, timepoints))
+
+    frametimes = timestep * np.arange(timepoints)
+    X = _full_rank(_cosine_drift(128, frametimes))[0]
```

Comment:
this number appears to be coming from the spm default high-pass filter. if this is indeed necessary shouldn't the high-pass filter cutoff be a variable?